### ISSUE TITLE
Alternative

### ISSUE THREAD

**Issue by evancohen** [noffle/electron-speech#10]
I've been working on an alternative that is written in Node, uses Google Cloud Speech, has offline hotword detection, and is compatible with Electron. Would you mind linking to it from the README?

https://github.com/evancohen/sonus/

I'd be happy to send a PR too 🚀

**Comment by hackergrrl** [comment]
Sonus looks excellent, Evan! Really really nicely done. `snowboy` looks
great too: I hadn't seen it before!

Please do send a PR my way for this. Very excited about your work.

**Comment by hackergrrl** [comment]
I'm curious about licensing. Sonus is permissively licensed, but snowboy has a restrictive one. My licensing knowledge is limited, but doesn't Sonus inherit snowboy's restrictions?

**Comment by evancohen** [comment]
Sorry for the super long delay, I've been busy. My licencing knowledge is also pretty limited... My understanding is that because Sonus doesn't actually distribute Snowboy code/IP it does not inherit Snowboy's license. BUT, should you choose to commercialize a piece of software/hardware that depends on Sonus (and therefore Snowboy) they you would have to adhere to the licence of both.

tl;dr so long as you don't sell what you are building there's nothing to worry about. If you do then you should get a lawyer.
